assets/app/scripts/misc/labelSelector.js

```diff
@@ -107,6 +117,11 @@ LabelSelector.prototype.hasConjunct = function(conjunct) {
 
 // Test whether this label selector covers the given selector
 LabelSelector.prototype.covers = function(selector) {
+  // "null" label selectors don't match anything
+  if (!this._emptySelects && this.isEmpty()) {
+    return false;
+  }
```

Comment:
same comment as above about returning true or false. does an empty label selector cover another empty label selector? what if their `_emptySelectsAll` flag is different?